packages/io/amf-deserializer/deserialize.js

```diff
@@ -1,27 +1,28 @@
-const {CSG} = require('@jscad/csg')
 
 const createObject = require('./objectBuilder')
+
 const parse = require('./parse')
 
-const deserializeToCSG = function (src, filename, options) {
-  filename = filename || 'amf'
-  const defaults = {pxPmm: require('./constants').pxPmm, version: '0.0.0', addMetaData: true}
+const instantiate = function (src, filename, options) {
+  const defaults = {
+    pxPmm: require('./constants').pxPmm
```

Comment:
hmm I think you removed version & addMetadata from the defaults because they are not used here right ?
I think the initial idea was to keep them around to have a consistent API between parsers:
- so this could either indicate that we are missing some output here ?
- or the are really useless here ?

Reply:
See next comment below.